src/getGroupRDF.js

```diff
@@ -1,10 +1,22 @@
-const fs = require('fs');
-const path = require('path');
-const SinopiaServer = require('sinopia_server')
+// Copyright 2019 Stanford University see LICENSE for license
 
+import fs from 'fs'
+import config from 'config'
+import SinopiaServer from 'sinopia_server'
 
-instance = new SinopiaServer.LDPApi()
-instance.apiClient.basePath = 'https://trellis.sinopia.io'
+
+var clientInstance = null
+
+// lazy instantiation of the client makes mocking its behavior easier
+export const sinopiaClient = () => {
+  if(!clientInstance) {
+    clientInstance = new SinopiaServer.LDPApi()
+    clientInstance.apiClient.basePath = config.get('trellis.basePath')
+    console.debug(`Sinopia Server client lazily instantiated.  base URL: ${clientInstance.apiClient.basePath}`)
```

Comment:
Cruft?

Reply:
hmm... despite it being `debug`, that was intentional, just so the user would see clearly what server they're hitting (since it's otherwise buried in a config file).

but happy to remove if that's overkill.  i had a bit more logging of that sort that i removed because i ended up feeling like it was unnecessary noise.